lib/gherkin_lint/linter/required_tags_starts_with.rb

```diff
@@ -0,0 +1,20 @@
+require 'gherkin_lint/linter/tag_constraint'
+require 'gherkin_lint/linter'
+
+module GherkinLint
+  # service class to lint for tags used multiple times
+  class RequiredTagsStartsWith < Linter
+    include TagConstraint
+
+    def match_pattern?(target)
+      match = false
```

Comment:
you mean the ```match``` var?  Can you tell we are both Java programmers?  I can try to make this more Rubyesque